.travis.yml

```diff
@@ -111,19 +111,39 @@ install:
     fi
   - |
     if [[ $TRAVIS_PHP_VERSION != "nightly" && $TRAVIS_PHP_VERSION != "8.0" ]]; then
-      travis_wait composer install --no-interaction --no-progress --no-scripts --no-suggest --optimize-autoloader --prefer-dist --verbose
+      travis_wait composer install \
+        --no-interaction \
+        --no-progress \
+        --no-scripts \
+        --no-suggest \
+        --optimize-autoloader \
+        --prefer-dist --verbose
     else
       composer require --no-update --no-suggest --no-scripts phpcompatibility/php-compatibility
-      travis_wait composer install --no-dev --no-interaction --no-progress --no-scripts --no-suggest --optimize-autoloader --prefer-dist --verbose --ignore-platform-reqs
+      travis_wait composer install \
+        --ignore-platform-reqs \
+        --no-dev \
+        --no-interaction \
+        --no-progress \
+        --no-scripts \
+        --no-suggest \
+        --optimize-autoloader \
+        --prefer-dist \
+        --verbose
     fi
-  - |
+  - >
```

Comment:
Is this correct ?

Reply:
It'll _work_ but I think you are right that it is not _correct_ (i.e. could lead to trouble later). I'll add a fix before tagging and including it in the upcoming minor release.